clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/provider/agent/AbstractAppengineCachingAgent.groovy

```diff
@@ -40,6 +40,26 @@ abstract class AbstractAppengineCachingAgent implements CachingAgent, AccountAwa
     this.credentials = credentials
   }
 
+  boolean shouldIgnoreLoadBalancer(String loadBalancerName) {
+    if (credentials.services != null && !credentials.services.contains(loadBalancerName)) {
```

Comment:
In terms of default behavior, I would recommend reading empty and null as "not specified", especially since it's easy to either have halyard or another config gen tool generate empty as the default value, and then it's also possible that a change in the default config properties leaves those lists with an empty array as the default.